alicloud/service_alicloud_cen.go

```diff
@@ -339,6 +339,64 @@ func (client *AliyunClient) WaitForCenInterRegionBandwidthLimitDestroy(cenId str
 	return nil
 }
 
+func (client *AliyunClient) DescribePublishedRouteEntriesById(paras CenRouteEntryPublishmentParas) (c cbn.PublishedRouteEntry, err error) {
```

Comment:
If the id contains `cen_id`, I think the method can use directly the resource ID to fetch the route entry.